src/ChemistryFeaturization.jl

```diff
@@ -46,10 +48,10 @@ and "magical decoding" bits.
 
 Representative examples – feature: feature object type, input to encoder, output of encoder:
 
-block (s,p,d,f):            AtomFeature (contextual=false), String, Flux.OneHotMatrix 
-electronegativity (binned): AtomFeature (contextual=false), Float32, Flux.OneHotMatrix
-electronegativity (direct): AtomFeature (contextual=false), Float32, Vector{Float32}
-oxidation state:            AtomFeature (contextual=true) Int, Vector{Float32}
+block (s,p,d,f):            AtomFeatureDescriptor (contextual=false), String, Flux.OneHotMatrix
```

Comment:
we should get rid of `contextual=false` in all of these since that field no longer exists ;)

Reply:
This should also probably be updated to be `ElementFeatureDescriptor` or `SpeciesFeatureDescriptor` in lieu of `AtomFeatureDescriptor` too.

Reply:
Resolved in 43bd2f9.